server/libs/infra/src/repositories/album.repository.ts

```diff
@@ -123,11 +123,31 @@ export class AlbumRepository implements IAlbumRepository {
     });
   }
 
-  create(album: Partial<AlbumEntity>): Promise<AlbumEntity> {
+  async hasAsset(id: string, assetId: string): Promise<boolean> {
+    const count = await this.repository.count({
```

Comment:
Why count instead of exists?

Reply:
To me, `exist` implies a boolean, where this is a number, so I went with count. I don't have strong opinions about it though.

Reply:
I meant for the `repository.count` instead of `repostitory.exist`, should've said that clearer. But either way, it doesn't really matter

Reply:
Oh. For that, I did not know it existed :smile:. I'll update it in my next PR.